### PR TITLE
Fix example annotations

### PR DESCRIPTION
**Pull request by thesadru**
## Summary

Partway to annotating everything

The param examples will be all changed in the rewrite so that one will be left as it is.

## Checklist

- [ ] If code changes were made then they have been tested
    - [ ] I have updated the documentation to reflect the changes
    - [x] I have formatted the code properly by running `black .`
- [ ] This PR fixes an issue
- [ ] This PR adds something new (e.g. new method or parameters)
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [x] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: examples/basic_voice.py [hunk 1]
```diff
@@ -112,9 +115,6 @@ async def stop(self, ctx):
 
         await ctx.voice_client.disconnect()
 
-    @play.before_invoke
-    @yt.before_invoke
-    @stream.before_invoke
```
Comment (EQUENOS) [inline examples/basic_voice.py:117]: This example was about pre-invoke hooks, now it's about something else

Comment (thesadru) [reply examples/basic_voice.py:117]: I have no idea what the fuck is going on in the hook decorators but it was complaining so I just yeeted it.

File: examples/converters.py [hunk 2]
```diff
@@ -40,51 +40,6 @@ async def userinfo_error(ctx: commands.Context, error: commands.CommandError):
...
-@bot.command()
-async def notify(ctx: commands.Context, target: ChannelOrMemberConverter):
-    # This command signature utilises the custom converter written above
-    # What will happen during command invocation is that the `target` above will be passed to
-    # the `argument` parameter of the `ChannelOrMemberConverter.convert` method and
-    # the conversion will go through the process defined there.
-
-    await target.send(f"Hello, {target.name}!")
-
-
```
Comment (EQUENOS) [inline examples/converters.py:87]: I understand that you don't like converters but they still exist

Comment (thesadru) [reply examples/converters.py:87]: Good fucking luck annotating that shit tho. I'd remove them purely out of that reason.

Comment (EQUENOS) [reply examples/converters.py:87]: Just don't touch the existing converters, some people are still using them

Comment (thesadru) [reply examples/converters.py:87]: Their behavior won't change. This is just for cleaning up the examples so we can eventually get rid of all linter errors over the entire project.

**Pull request merged**